### ISSUE TITLE
unknown instruction: CD

### ISSUE THREAD

**Issue by bovesan** [adamrehn/ue4-docker#215]
This is already fixed in 812e0c0b6ff7cfe5711335e89e13fb1fa582fd44, but is awaiting release, which means it's still broken if you follow the [official instructions](https://docs.adamrehn.com/ue4-docker/configuration/configuring-windows-server).
## Symptom
The Windows ue4-source image fails with this:
```
...
Error response from daemon: dockerfile parse error line 30: unknown instruction: CD                                     
[ue4-docker build] Error: failed to build image "adamrehn/ue4-source:4.27.2-ltsc2022-vs2017". 
```
## Workaround
See @adamrehn's comment [below](#issuecomment-1005265555)
~~Replace `C:\Users\spacewalk\AppData\Local\Programs\Python\Python*\Lib\site-packages\ue4docker\dockerfiles\ue4-source\windows\Dockerfile` with [the updated version](https://raw.githubusercontent.com/adamrehn/ue4-docker/812e0c0b6ff7cfe5711335e89e13fb1fa582fd44/ue4docker/dockerfiles/ue4-source/windows/Dockerfile).
You can run this in a Powershell:
```wget https://raw.githubusercontent.com/adamrehn/ue4-docker/812e0c0b6ff7cfe5711335e89e13fb1fa582fd44/ue4docker/dockerfiles/ue4-source/windows/Dockerfile -OutFile $home\AppData\Local\Programs\Python\Python*\Lib\site-packages\ue4docker\dockerfiles\ue4-source\windows\Dockerfile```~~

## Reference
Output of the `ue4-docker info` command:
```
ue4-docker version:         0.0.94 (latest available version is 0.0.94)
Operating system:           Windows Server 2022 Datacenter Azure Edition (Build 20348.405)
Docker daemon version:      20.10.7
NVIDIA Docker supported:    No
Maximum image size:         1000GB
Available disk space:       991.35 GiB
Total system memory:        32 GiB physical, 5 GiB virtual
Number of processors:       4 physical, 8 logical
```

**Comment by adamrehn** [comment]
Just for reference, there's a far simpler way to install the latest version of ue4-docker from the [master](https://github.com/adamrehn/ue4-docker/tree/master) branch without needing to cherrypick individual files:

```
pip3 install git+https://github.com/adamrehn/ue4-docker.git
```

This uses [pip's built-in VCS support](https://pip.pypa.io/en/stable/topics/vcs-support/) to clone and install the latest source code from GitHub. You can run this command whenever there are unreleased fixes that you want to use locally.

**Comment by slonopotamus** [comment]
I've just released ue4-docker v0.0.95 that includes 812e0c0b6ff7cfe5711335e89e13fb1fa582fd44.
